src/pages/expansions/StocktakeEditExpansion.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2016
+ */
+
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import autobind from 'react-autobind';
+import { Expansion } from 'react-native-data-table';
+import { dataTableStyles, expansionPageStyles } from '../../globalStyles';
+import { PageButton, ExpiryTextInput, PageInfo } from '../../widgets';
+import { GenericPage } from '../GenericPage';
+import { parsePositiveInteger } from '../../utilities';
+import { tableStrings, buttonStrings } from '../../localization';
+
+/**
+* Renders page to be displayed in StocktakeEditPage -> expansion.
+* @prop   {Realm}               database      App wide database.
+* @state  {Realm.Results}       items         the stocktakeItems of props.stocktake.
```

Comment:
This isn't actually in state. Shouldn't it be just one stocktakeItem? Capitalise 'the'.

Would be good to explain props.data and props.refreshParent.